parser_test.go

```diff
@@ -548,7 +549,8 @@ const expectedListOutput = `
 	"status_superserver": ["10.120.0.226"],
 	"database": ["10.0.0.8"],
 	"scw_test": ["10.0.0.11"],
-	"tfinventory_rocks": ["10.0.0.19"]
+	"tfinventory_rocks": ["10.0.0.19"],
+    "superman_clarkkent": ["10.0.0.19"]
```

Comment:
Please fix indentation. This file uses tabs.

Reply:
Good catch, thanks! It's fixed now. :)